tests/composer.json.travis

```diff
@@ -0,0 +1,170 @@
+{
+  "name": "PrestaShop/PrestaShop",
+  "description": "PrestaShop offers a free, fully scalable, Open Source e-commerce solution.",
+  "type": "project",
+  "repositories": [
+    {"type": "vcs", "url": "https://github.com/prestashop/blockcart"},
```

Comment:
@maximebiloe it's [composer](https://getcomposer.org/doc/05-repositories.md#vcs) :)